core/composer.js

```diff
@@ -188,3 +191,7 @@ export function getFinishedComposes() {
 export function getFailedComposes() {
   return get("/api/v0/compose/failed").then(data => data.failed);
 }
+
+export function getComposeLog(uuid) {
+  return get("/api/v0/compose/log/" + encodeURIComponent(uuid), undefined, { format: "raw" });
```

Comment:
@larskarlitski that's why I had all args to get to be named args before, to avoid this undefined variable here.